Backpack-SwiftUI/Rating/Classes/BPKRating.swift

```diff
@@ -0,0 +1,129 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKRating: View {
+
+    private let accessibilityLabel: String
+    private let value: Float
+    private let ratingScale: Scale
+    private let size: Size
+    private let subtitle: String?
+    private let showScale: Bool
+    @ViewBuilder private var titleViewBuilder: () -> AnyView
+
+    public var body: some View {
+        HStack(alignment: .lastTextBaseline, spacing: BPKSpacing.md.value) {
+            HStack(alignment: .lastTextBaseline, spacing: .zero) {
+                BPKText(
+                    ratingScale.displayedValue(from: value),
+                    style: size.fontStyle.ratingValueLabelFontStyle
+                ).foregroundColor(.textPrimaryColor)
```

Comment:
BPKText has a default color of TextPrimary, we probably don't want to set it here. (If the default ever changes)

Reply:
Fixed in 51568879d49f2894f41e2bd9bce6b544117de349